docker/build_testing_sheet.dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:bionic
+
+RUN apt-get update -y
+
+COPY . /kolibri
+
+COPY ./kolibri/.buildkite $GOOGLE_SPREADSHEET_CREDENTIALS
```

Comment:
Could you clarify what this is meant to do?

Reply:
The integration testing script need the Google api credentials `json` file which is the path of it is in the `GOOGLE_SPREADSHEET_CREDENTIALS ` environment variable to be able to access the script I'll copy it inside the docker environment.

Reply:
My understanding, though, is that you're going to copy the contents of the `.buldkite` directory into the _container_, at a directory specified in the environment variable `$GOOGLE_SPREADSHEET_CREDENTIALS`

Reply:
I see what you mean. I'll fix it thanks.